Lab Member YAMLs/Templates/Full Name with Spaces.yaml

```diff
@@ -0,0 +1,46 @@
+name:
+birthday: MM.DD
+position: (Job Title) (Graduate Student) (Postoctoral Research Scholar/Fellow)
+affiliations: >
+  Weill Cornell Physiology, Biophysics, and Systems Biology PhD Program
+  Weill Cornell Biochemistry and Structural Biology PhD Program
+  Tri-Institutional PhD Program in Computational Biology and Medicine
+  Tri-Institutional PhD Program in Chemical Biology
+  Louis V. Gerstner, Jr. Graduate School of Biomedical Sciences
+  Berlin Institute of Health (BIH) Einstein Foundation - Stiftung Charité Group
+  Joint Mentorship with Name (URL to lab)
+fellowships: >
```

Comment:
Same here: This should be a list.
Maybe it should be `awards`? 

We can use comments to explain how folks should fill this in:
```yaml
# List all awards and fellowships here
awards:
  - Ig Nobel Prize in Economics
  - Lithuanian Horticultural Society Outstanding Mentorship Award
  - NSF GRFP Graduate Fellowship
  - American Philatelic Society Poster Prize
```